scrapi/database.py

```diff
@@ -16,13 +16,47 @@
 logger = logging.getLogger(__name__)
 
 
-try:
-    cqlengine.connection.setup(settings.CASSANDRA_URI, settings.CASSANDRA_KEYSPACE)
-    management.create_keyspace(settings.CASSANDRA_KEYSPACE, replication_factor=1, strategy_class='SimpleStrategy')
-except NoHostAvailable:
-    logger.error('Could not connect to Cassandra, expect errors.')
-    if settings.RECORD_HTTP_TRANSACTIONS or 'cassandra' in settings.NORMALIZED_PROCESSING or settings.RAW_PROCESSING:
-        raise
+class DatabaseManager(object):
```

Comment:
_DatabaseManager (if you don't want people to be defining new managers outside of the module)